src/diet.ts

```diff
@@ -982,3 +1007,22 @@ export function runDiet(): void {
     }
   });
 }
+
+export function runNightcap(): void {
+  if (getRemainingLiver() === 0) {
+    withVIPClan(() => {
+      const dietBuilder = computeDiet();
+
+      if (have($familiar`Stooper`)) {
+        if (myFamiliar() !== $familiar`Stooper`) {
+          useFamiliar($familiar`Stooper`);
```

Comment:
Does this take into account case of someone who (for some reason) has but cannot use Stooper? (in post-ronin g-lover for instance) [admittedly no idea why someone would be running garbo in that case, but want to check if its something we want to care about]

Reply:
yes - this will attempt to use the stooper, and if it can't be used it will just continue to nightcap